astropy/io/fits/card.py

```diff
@@ -24,6 +24,7 @@
 KEYWORD_LENGTH = 8  # The max length for FITS-standard keywords
 
 VALUE_INDICATOR = '= '  # The standard FITS value indicator
+VALUE_INDICATOR_LEN = 2
```

Comment:
Wouldn't it be more semantically correct to replace the `2` with `len(VALUE_INDICATOR)`?

Reply:
Yes, but I hard coded the value to avoid a tiny overhead at import time.

Reply:
Is it worth it?

Reply:
The overhead is probably negligible, I can change if you prefer. But the value is hardcoded and defined just above so using `len` seems overkill.

Reply:
Maybe overkill, but I only commented because it took me a few seconds to figure out what this value is (probably more time than saved by hardcoding it 😄).

Reply:
Thanks for changing it 👍